skbio/stats/tests/test_composition.py

```diff
@@ -45,10 +48,73 @@ def setUp(self):
         self.rdata1 = [[0.70710678, -0.70710678, 0., 0.],
                        [0.40824829, 0.40824829, -0.81649658, 0.],
                        [0.28867513, 0.28867513, 0.28867513, -0.8660254]]
+        self.table1 = np.array([
```

Comment:
Can you name the valid and invalid data more descriptively so that it's clear what is being tested? Some comments would also help.